include/platform.h

```diff
@@ -1,6 +1,7 @@
 #ifndef PLATFORM_H
 #define PLATFORM_H
 #include <SFML/Graphics.hpp>
+#include <SFML/System.hpp>
```

Comment:
Why is this needed?

Reply:
Not sure, I didn't add these...it's in a couple of header files though

Reply:
@carter-greis do you know?

Reply:
I would just remove it and if it compiles fine, then it's not needed. I'm pretty sure it's for handling window properties